tests/src/_config.scss

```diff
@@ -1,3 +1,4 @@
+$font-size-root: 18px;
```

Comment:
Why 18px and not 16px ?
The "$spaces-base" is based with 16px, and 16px is the font-size root of many website.

Reply:
It is a test file, the value isn't important.

Reply:
@w-jerome It's in the test configuration, the default value is 16px, see the line 9 in [`src/framework/_typography.scss`](https://github.com/studiometa/scss-toolkit/pull/37/files#diff-3893b586207d60bc7142708a61f1558e):

```scss
/** @type {String} Root font size, used to calculate others font sizes */
$font-size-root: 16px !default;
```